ddtrace/bootstrap/sitecustomize.py

```diff
@@ -43,6 +43,22 @@ def update_patched_modules():
         EXTRA_PATCHED_MODULES.update({module: should_patch.lower() == 'true'})
 
 
+def add_global_tags(tracer):
+    tags = {}
+    for tag in os.environ.get('DD_TRACE_TAGS', '').split(','):
+        if len(tag.split(':')) != 2:
+            log.debug("skipping malformed tracer tag")
+            continue
+
+        tag_name, tag_value = tag.split(':')
```

Comment:
Not a required change, but we could use `.partition(':')` here instead of the `len()` check and this line.

e.g. 

```python
tag_name, _, tag_value = tag.partition(':')
if not tag_name or not tag_value:
    log.debug('skipping malformed tracer tag, {}'.format(tag))
    continue
```

No strong preference, just pointing out.

Reply:
I share your preference! Always a good day to use `partition`.